### PR TITLE
initial gui tools boilerplate

### PR DESCRIPTION
**Pull request by dakom**
First in a series of PRs that will build on eachother towards https://github.com/Lay3rLabs/avs-toolkit/issues/48

This one just has the basic boilerplate to work and connect in a browser with Keplr

Had to add some new dependencies and upgrade `climb` which had a one-liner effect on some other code (`parse_address_value` -> `parse_address_str`)

My hope is that once this one is in, the rest will only touch the GUI directory

### CODE REVIEW

**Review by ueco-jb — APPROVED** [review]
A huge amount of work, very nice

**Comment by dakom** [comment]
> A huge amount of work, very nice

Thanks! It's copy/pasted from some other projects (like climb itself), but it is the culmination of lots of stuff over time :)

File: tools/gui/src/atoms/checkbox.rs [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
...0552 0.447567 4.96974 0.445786 6.60129V34.2936C0.447567 35.9252 1.09649 37.4894 2.25017 38.6431C3.40386 39.7967 4.96808 40.4457 6.59963 40.4474H34.2919C35.9235 40.4457 37.4877 39.7967 38.6414 38.6431C39.7951 37.4894 40.444 35.9252 40.4458 34.2936V6.60129C40.444 4.96974 39.7951 3.40552 38.6414 2.25184C37.4877 1.09815 35.9235 0.44923 34.2919 0.447449ZM30.8544 13.7446L17.9314 29.1292C17.7896 29.298 17.6132 29.4344 17.4141 29.5292C17.2151 29.6239 16.998 29.6747 16.7775 29.6782H16.7516C16.5359 29.6781 16.3227 29.6327 16.1257 29.5449C15.9288 29.4571 15.7525 29.3289 15.6083 29.1686L10.0698 23.0148C9.92917 22.8656 9.81975 22.6898 9.748 22.4977C9.67626 22.3056 9.64363 22.1011 9.65204 21.8963C9.66045 21.6914 9.70973 21.4903 9.79697 21.3047C9.88422 21.1192 10.0077 20.9529 10.1601 20.8158C10.3125 20.6786 10.4908 20.5733 10.6845 20.5061C10.8782 20.4388 11.0833 20.4109 11.288 20.4241C11.4926 20.4372 11.6925 20.4912 11.876 20.5827C12.0595 20.6742 12.2228 20.8015 12.3564 20.9571L16.7112 25.7955L28.4987 11.7657C28.7631 11.46 29.1372 11.2707 29.5401 11.2386C29.943 11.2064 30.3423 11.3342 30.6518 11.5941C30.9613 11.8541 31.156 12.2254 31.1939 12.6278C31.2319 13.0302 31.1099 13.4314 30.8544 13.7446Z",
```
[the remark this answers is not in the document]
Comment (dakom) [reply tools/gui/src/atoms/checkbox.rs:85]: This is just dominator syntax for the markup

Other frameworks, like React, do the same thing but they use JSX to make it look more like HTML

You have to do it this way if you want to be able to change properties at runtime (well, you could also hardcode identifiers in the SVG file and then get at the elements from the outside in, sorta, but that's a bit finicky)

File: tools/gui/config.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+    "chains": {
```
Comment (ismellike) [inline tools/gui/config.json:2]: We should probably update the config.json to match [this](https://github.com/Lay3rLabs/avs-toolkit/blob/main/tools/cli/config.json)

Comment (ismellike) [reply tools/gui/config.json:2]: Even better, we should use the same config.json for cli and gui

Comment (dakom) [reply tools/gui/config.json]: good call, consolidated into one config file in https://github.com/Lay3rLabs/avs-toolkit/pull/74/commits/3fcff82696b63897dc83e85cc9c7f0543e74faf3

**Pull request merged**